app/javascript/components/base/dots-dropdown.vue

```diff
@@ -66,5 +69,15 @@ export default {
       this.toggleDropdown();
     },
   },
+  computed: {
+    dropdownStyle() {
+      if (!this.elements.last) {
+        return '-left-20 absolute mt-0.5 w-32 bg-white border overflow-hidden rounded-lg shadow-md cursor-pointer z-10';
+      }
+
+      // eslint-disable-next-line max-len
+      return '-left-20 bottom-10 absolute mt-0.5 w-32 bg-white border overflow-hidden rounded-lg shadow-md cursor-pointer z-10';
```

Comment:
🚫 **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>This line has a length of 128. Maximum allowed is 120  vue/max-len